House-of-glass/src/components/Filters/FilterCompose.jsx

```diff
@@ -0,0 +1,56 @@
+
+
+const hightolow =(state,array)=>{
+    let sortedList = state.sortBy==="HIGHTOLOW"?[...array].sort((a,b)=>Number(b.price)- Number(a.price)):[...array];
```

Comment:
sortBy is not a pure function hence it alters value of state. 
That makes your hightolow function impure. Would advise to change it to a pure function.
Text me if you need detailed explanation as to how this may bring about unwanted bugs